auction-server/src/auction.rs

```diff
@@ -1340,6 +1355,21 @@ impl ChainStore for ChainStoreSvm {
     const CHAIN_TYPE: models::ChainType = models::ChainType::Svm;
     const AUCTION_MINIMUM_LIFETIME: Duration = Duration::from_millis(400);
 
+    async fn get_next_trigger<'a>(
```

Comment:
is it possible to extract slot number from this block trigger?
We need to know slot number to make sure running the conclusion loop for svm every 150 slots.